src/main/ui/chat.shader

```diff
@@ -0,0 +1,47 @@
+shader_type canvas_item;
+
+const float PI = 3.14159265358979;
+const vec2 sprite_size = vec2(2048.0, 512.0);
+
+uniform vec4 border_color : hint_color;
+uniform vec4 center_color : hint_color;
+uniform vec4 accent_color : hint_color;
+
+uniform sampler2D accent_texture;
```

Comment:
Make sure to document the requirements for this source texture, because they're sort of baffling.

Reply:
`accent_texture` is pretty straightforwardly a monochrome texture. However I've gone into detail about the shader's target texture which has stranger requirements.